docs/FAQ.md

```diff
@@ -1,15 +1,159 @@
+4. [Why does TrenchBoot use an intermediate launcher?](
+    #4-why-does-trenchboot-use-an-intermediate-launcher)
+5. [What are the benefits of measurement over signature validation?](
+    #5-what-are-the-benefits-of-measurement-over-signature-validation)
+6. [What do I need to incorporate TrenchBoot into my system?](
+    #6-what-do-i-need-to-incorporate-trenchboot-into-my-system)
+7. [Where do I start if I want to help with contributions?](
+    #7-where-do-i-start-if-i-want-to-help-with-contributions)
+8. [Where to find the sources?](#8-trenchboot-sources)
+9. [Various GitHub and further documentation links](
+    #9-various-github-and-further-documentation-links)
 
-## 1. Why does TrenchBoot use an intermediate launcher?
+## 1. Why use Trenchboot? (background info)
+
+### What is Trust?
+
+Trust is when the computer does something on our behalf and we have to trust it
+with our data. A trusted component is that which always behaves in an expected
+manner.  To make a component "trusted", you must establish some root of trust.
+A Root of Trust Measurement is the process of measuring (hashing) components on
+a system to validate their integrity. When establishing a root of trust, there
+are two primary schools of thought: Static Root of Trust Measurements and
+Dynamic Root of Trust Measurements.
+
+### SRTM
+
+SRTM takes place at system boot from firmware / bios. This process requires
+measurements of the entire boot sequence at once. This is both a good and a
+bad thing, because any changes to any part of the process (including even
+bios settings) would cause the entire system to need to be remeasured. SRTM
+Starts with Core Root of Trust for Measurement (CRTM), and works its way up the chain where
+each link verifies the next. In other words, SRTM starts by measuring the bios
+boot block and then moves on to measure the rest of the bios.
+
+### DRTM
+
+DRTM aims to address shortcomings of SRTM. DRTM creates trusted environment from
+an untrusted state. Essentially what this means is that regardless of weather
+or not computer was booted from trusted software, it uses hardware to ensure
+that no malicious software has been run. For this reason, DRTM takes place
+after boot, as opposed to SRTM, which takes place pre-boot. Intel's solution
+for DRTM is TXT, which utilizes special instructions on the processor (SMX).
+AMD's solution for DRTM is called SKINIT. Trenchboot provides a solution for
+setting up DRTM for an arbitrary platform.
+
+### Typical Measured Launch Process
+
+The process for carrying out DTRM depends on your system:
+
+- [Intel TXT: Software Development Guide][txt-spec]
+- [System Programming Volume of for AMD64 (PDF)][skinit-spec] (see
+  "Secure Startup with SKINIT" section)
+
+[txt-spec]: https://www.intel.com/content/www/us/en/content-details/315168/intel-trusted-execution-technology-intel-txt-software-development-guide.html
+[skinit-spec]: https://www.amd.com/system/files/TechDocs/24593.pdf
+
+### So, why TrenchBoot?
```

Comment:
Drop the So - just Why TrenchBoot

Edited-by: Amanda Ferguson [af.jenks@gmail.com](mailto:af.jenks@gmail.com)